tests/test_iam/test_iam.py

```diff
@@ -2879,3 +2880,450 @@ def test_delete_role_with_instance_profiles_present():
     role_names = [role["RoleName"] for role in iam.list_roles()["Roles"]]
     assert "Role1" in role_names
     assert "Role2" not in role_names
+
+
+@mock_iam
```

Comment:
Add in a test case with `@mock_config` where you test that the AWS Config APIs themselves properly return a correct IAM object back out.  It doesn't have to be too thorough, but it basically tests that the front end and backend logic all works together.

Reply:
added this test, as well as some additional "test writing criteria" in the config readme to cover all the bases